### PR TITLE
feat(php.net): add presence

### PR DESCRIPTION
**Pull request by iLazlow**
**Describe the changes in this pull request:**
This pull requests adds a presence for the php.net website
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

</details>

### CODE REVIEW

**Comment by DarkVillager** [comment]
**Instead of:**
poc_php_docs_1 
**use:**

File: websites/P/PHP.net/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	"description": {
+		"en": "The official website and documentation for PHP."
+	},
+	"url": [
+		"php.net",
+		"www.php.net"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/cC2n2S6.png",
+	"thumbnail": "https://i.imgur.com/O0eMrCB.png",
```
Comment (EncryptedDev) [inline websites/P/PHP.net/dist/metadata.json:17]: This is not big enough to be a thumbnail

File: websites/P/PHP.net/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.6",
+	"author": {
+		"name": "iLazlow",
+		"id": "132835232289259520"
+	},
+	"service": "PHP.net",
+	"description": {
+		"en": "The official website and documentation for PHP."
```
Comment (EncryptedDev) [inline websites/P/PHP.net/dist/metadata.json:9]: Not the official description for php

File: websites/P/PHP.net/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+		"php.net",
+		"www.php.net"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/cC2n2S6.png",
+	"thumbnail": "https://i.imgur.com/O0eMrCB.png",
+	"color": "#8892BF",
+	"category": "other",
+	"tags": [
+		"other",
```
Comment (EncryptedDev) [inline websites/P/PHP.net/dist/metadata.json:21]: I don't see the point in this

File: websites/P/PHP.net/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+		"development",
+		"php-net",
+		"php",
```
Comment (EncryptedDev) [inline websites/P/PHP.net/dist/metadata.json:24]: Irrelevant tags too

File: websites/P/PHP.net/presence.ts [hunk 5]
```diff
@@ -0,0 +1,66 @@
+const presence = new Presence({
+		clientId: "952575137180442626"
+	}),
+	browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (EncryptedDev) [inline websites/P/PHP.net/presence.ts:4]:
```suggestion
	browsingTimestamp = Math.floor(Date.now() / 1000);
```

File: websites/P/PHP.net/presence.ts [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+presence.on("UpdateData", () => {
+	const data: PresenceData = {
```
Comment (EncryptedDev) [inline websites/P/PHP.net/presence.ts:7]: use `presenceData` instead

File: websites/P/PHP.net/tsconfig.json [hunk 7]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist/"
+  }
+}
```
Comment (EncryptedDev) [inline websites/P/PHP.net/tsconfig.json:6]: You should format with tabs

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Hi, thank you for opening this pull request. I have requested some changes found below: 

Reach out to me on Discord if you need any further help

**Review by iLazlow — COMMENTED** [review]
I've changed the requested files in my last commit. Please let me know if there is anything wrong.

**Review by EncryptedDev — DISMISSED** [review]

File: websites/P/PHP.net/presence.ts [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "php_logo",
+			startTimestamp: browsingTimestamp
+		},
+		route = document.location.pathname.split("/");
+	presenceData.buttons = [
+		{ label: "View Website", url: document.location.href }
+	];
```
Comment (Slowlife01) [inline websites/P/PHP.net/presence.ts:14]: Remove button on the main page

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**